lib/fizz_buzz.rb

```diff
@@ -0,0 +1,16 @@
+class FizzBuzz
+
+  def self.divisible_by(number)
+    if number == ""
+      0
+    elsif number.to_i % 15 == 0
+      "Fizz Buzz"
+    elsif number.to_i % 3 == 0
+      "Fizz"
+    elsif number.to_i % 5 == 0
+      "Buzz"
+    else number.to_i
```

Comment:
For clarity, the return statement for the else should be on the next line, just as all your other conditions' return statements are.
e.g.

if "this is the case"
  do this
elsif "this is the other case"
 do this other thing
else
  do something completely different